mkdocs/livereload/__init__.py

```diff
@@ -30,6 +30,7 @@
 var livereload = function(epoch, requestId) {
     var req = new XMLHttpRequest();
     req.onloadend = function() {
+        window.removeEventListener("beforeunload", abort)
```

Comment:
Let's do semicolons consistently

```suggestion
        window.removeEventListener("beforeunload", abort);
```